build-logic/convention/build.gradle.kts

```diff
@@ -58,59 +58,59 @@ tasks {
 gradlePlugin {
     plugins {
         register("androidApplicationCompose") {
-            id = "nowinandroid.android.application.compose"
+            id = libs.plugins.nowinandroid.android.application.compose.get().pluginId
```

Comment:
Can we also use these as the single source of truth for all `id` matchers used in the project?

Reply:
Which "matchers" are you referring to? In the `plugins {}` blocks?

Reply:
Since there has been no answer from @baksha97 I'm resolving this comment.